7.8.Вложенные_циклы._Часть_1/7.8.7.py

```diff
@@ -0,0 +1,6 @@
+n = int(input())   
+
+for i in range(1, n + 1):            
+    for j in range(1, 10):          
+        print(i, '+', j, '=', j + i)
```

Comment:
Можно так, но лучше f-строки:
```suggestion
        print(f"{i} + {j} = {i + j}")
```